qemu/tests/cfg/blockdev_stream_dirty_bitmap.cfg

```diff
@@ -29,3 +29,5 @@
         lun_data1 = 1
         enable_iscsi_data1sn = no
         image_raw_device_data1sn = no
+    ceph:
+        image_format_data1 = raw
```

Comment:
enable_ceph_data1sn = no

Reply:
Have updated, thanks zhenchao.

Reply:
this one should be enable_ceph_data1sn instead of enable_ceph_sn1